src/sparse13/spmatrix.h

```diff
@@ -50,7 +50,9 @@
 #include "spconfig.h"
```

Comment:
about above 8-9 files that git shows as empty files - what was changed?

Reply:
Must be a display bug. I've added some `extern "C"`